src/mesh/voxel_cloud.rs

```diff
@@ -14,6 +14,27 @@ use crate::plane::Plane;
 
 use super::{primitive, tools, Face, Mesh, NormalStrategy};
 
+/// Describes how should be empty (None) voxel be treated. Either as None or as
+/// a constant value. This is used for arithmetic functions, such as addition.
+#[allow(dead_code)]
+#[derive(Debug)]
+pub enum TreatEmpty {
+    AsNone,
+    AsValue(f32),
+}
+
+/// Selects falloff function for the falloff field computation. The parameter
+/// specifies a distance multiplier.
+#[allow(dead_code)]
+#[derive(Debug)]
+pub enum FalloffFunction {
+    Linear(f32),
+    Square(f32),
+    InverseLinear(f32),
+    InverseSquare(f32),
```

Comment:
I'd maybe call the inverse functions reciprocal. (reciprocal is multiplicative inverse, but there also is an additive inverse)

Reply:
It is a convention https://www.dpmag.com/how-to/tip-of-the-week/how-light-works-the-inverse-square-law/

Reply:
Ok, I don't feel strongly either way. Both are conventions in different fields it seems.